build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'java'
+apply plugin: 'war'
+
+repositories {
+    jcenter()
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencies {
+    compile fileTree(dir: 'lib', include: '*.jar')
+    compile 'javax.servlet:javax.servlet-api:3.0.1'
+    compile 'postgresql:postgresql:9.1-901.jdbc4'
+
+    testCompile 'simple-jndi:simple-jndi:0.11.4.1'
+    testCompile 'junit:junit:4.12'
+}
+
+sourceSets.main.java.srcDirs = ["src"]
+// the tests fail because they have environemnt specific parameters
+// sourceSets.test.java.srcDirs = ["test"]
```

Comment:
You are welcome. I've been using gradle for a while, let me know if I can help.
Have you ever considered publishing the project on bintray? At the moment I'm including the code directly in my build, but it would be more convenient to fetch the jar file.